src/elements/hx-tabset/index.spec.js

```diff
@@ -329,5 +329,32 @@ describe('<hx-tabset> component tests', () => {
             expect(firstTabCount).to.equal(0);
             expect(secoundTabCount).to.equal(1);
         });
+
+        it.skip(`[FEATURE] should Not able to Navigate between one or more tab panels in angular ${template}`, async () => {
```

Comment:
@BY00565233, please attach a screenshot of the test run.

Reply:
I have attached screenshot with new test case added,  please let me know if you want me to attach screen shot by removing skip

Reply:
Yes, please attach a screenshot by removing `skip`.

Reply:
Changes have been done, Please review